client/src/app/components/stardust/Unlocks.tsx

```diff
@@ -1,15 +1,30 @@
-import { ISignatureUnlock } from "@iota/iota.js-stardust";
+import { SIGNATURE_UNLOCK_TYPE, UnlockTypes } from "@iota/iota.js-stardust";
 import classNames from "classnames";
 import React, { useState } from "react";
+import { NameHelper } from "../../../helpers/stardust/nameHelper";
 import { ReactComponent as DropdownIcon } from "./../../../assets/dropdown-arrow.svg";
+import TruncatedId from "./TruncatedId";
 
 interface IUnlocksProps {
-    unlocks: ISignatureUnlock[];
+    unlocks: UnlockTypes[];
 }
 
 const Unlocks: React.FC<IUnlocksProps> = ({ unlocks }) => {
     const [isExpanded, setIsExpanded] = useState(false);
 
+    const displayUnlocksTypeAndIndex = (type: number, index: number) => (
+        <div>
+            <div className="unlocks-card--row">
+                <span className="label">Type:</span>
+                <span className="value">{NameHelper.getUnlockTypeName(type)}</span>
+            </div>
+            <div className="unlocks-card--row">
```

Comment:
Let's swap these two "cards" please. So index is first and then type, i think it will look better on UI that way.

Reply:
Done.